platform/linux-dpdk/pool/dpdk.c

```diff
@@ -91,12 +90,12 @@ int odp_pool_init_global(void)
 	return 0;
 }
 
-int odp_pool_init_local(void)
+static int dpdk_pool_init_local(void)
 {
```

Comment:
I am assuming this function is empty as DPDK already implements per core pool cache and we do not need to implement it ourselves.